docs/installation/requirements.md

```diff
@@ -4,146 +4,186 @@ sidebar_position: 1
-
-XCP-ng 8.2 requires an IPv4 network for management and storage traffic. Starting with XCP-ng 8.3, the management network can use IPv6.
+XCP-ng 8.2 requires an IPv4 network for management and storage traffic. Starting from XCP-ng 8.3, the management network supports IPv6.
 
 :::info
-Ensure that the time setting in the BIOS of your server is set to the current time in UTC. In some support cases, serial console access is required for debug purposes. When setting up XCP-ng configuration, we recommend that you configure serial console access. For hosts that do not have physical serial port or where suitable physical infrastructure is not available, investigate whether you can configure an embedded management device. For example, Dell DRAC or HP iLO. For more information about setting up serial console access, see [CTX228930 - How to Configure Serial Console Access on XenServer 7.0 and later](https://support.citrix.com/article/CTX228930).
+Set the server's BIOS clock to the current UTC time. For debugging support cases, serial console access may be required. Consider configuring serial console access for XCP-ng. For systems without physical serial ports, explore embedded management devices like Dell DRAC or HP iLO. See [CTX228930 - How to Configure Serial Console Access on XenServer 7.0 and later](https://support.citrix.com/article/CTX228930).
 :::
 
-## 🖥️ Supported guest OS
+## 🖥️ Supported Guest OSes
 
-We only officially support operating systems which still receive support from their publisher. But there's a lot more that can run on XCP-ng.
+XCP-ng officially supports operating systems still receiving updates from their publishers. Many other systems can also run on XCP-ng.
 
-### Windows based
+### Windows
 
-* Windows Server 2016, 2019, 2022
-* Windows 10
-* Windows 11 (starting with XCP-ng 8.3).
+- Windows Server 2016, 2019, 2022
+- Windows 10
+- Windows 11 (starting with XCP-ng 8.3)
 
 :::info
-Very old versions of Windows will run, but without PV drivers, so with lower networking and disk performance (Windows XP, Windows Server 2003, etc.).
-
-Less old but EOL versions of Windows, such as Windows Server 2012, currently run well but we don't offer guarantees for the future. For example, the future PV drivers updates might drop the compatibility with such old releases.
+Older Windows versions may work but lack PV drivers, resulting in lower networking and disk performance (e.g., Windows XP, Windows Server 2003). While unsupported versions like Windows Server 2012 still function, future PV driver updates may drop compatibility.
 :::
 
 ### Linux
 
-* RHEL, CentOS, Rocky, AlmaLinux, Oracle…
-* Debian & Ubuntu
-* Arch, Alpine, SUSE and many more…
+- RHEL, CentOS, Rocky, AlmaLinux, Oracle
+- Debian, Ubuntu
+- Arch, Alpine, SUSE, and others
 
 ### BSD
 
-* FreeBSD and related (pfSense, TrueNAS…)
-* OpenBSD
+- FreeBSD and related distributions (e.g., pfSense, TrueNAS)
+- OpenBSD
+
+## Virtual Machine Requirements
```

Comment:
I am not sure at all with this entire section:

* we are very cautious and a quick overview isn't great for us by looking at the numbers
* We tested more than 64 vCPU with some Linux VMs IIRC, and topology will be one important work next year. So if we provide a limit, we should always put that in perspective with actual work
* it's even worse for the 2TiB limit, which isn't true for all VDI. Raw are not limited, SMAPIv3 current drivers neither and we aren't far to get qcow2 on tapdisk. So to me it's simply too negative to restrict too much by displaying those numbers without context.
* I'm not aware of a CD drive limit per VM, I remember some XS doc explaining how to add another one

The rest of the PR is OK to me. Maybe we can start to merge everything except the VM limits which require more work to be suitable.
* I'm not even sure about some limits (only one passed through GPU? never heard of it)

Reply:
I tried to address all this in a new commit. We're not in a hurry to merge this PR so I think we can take the time to finalize it as a whole.